### ISSUE TITLE
Mod breaks lighting inside the ship

### ISSUE THREAD

**Issue by NineOfGaming** [sfDesat/Celestial-Tint#3]
Should be self explanatory, basically having this mod installed makes the ship interior have weird lighting

With Celestial Tint enabled
![Screenshot 2024-03-13 200103](https://github.com/sfDesat/Celestial-Tint/assets/94534207/7d4fe5dc-4967-402c-ad8d-f0bac43de860)
![Screenshot 2024-03-13 200112](https://github.com/sfDesat/Celestial-Tint/assets/94534207/5e574514-3866-447e-a13d-c114b97fafba)

With Celestial Tint disabled
![Screenshot 2024-03-13 200349](https://github.com/sfDesat/Celestial-Tint/assets/94534207/0875a687-4f17-495b-9c2e-041101e54154)
![Screenshot 2024-03-13 200357](https://github.com/sfDesat/Celestial-Tint/assets/94534207/a25fc3d0-80d8-4611-b460-4f0f4c084960)

**Comment by zndrmn** [comment]
This happens when ShipWindows `SpaceOutside` config is set to `1`
```
## Set this value to control how the outside space looks. (0 = Let other mods handle, 1 = Space HDRI Volume (default), 2 = Black sky with stars)
# Setting type: Int32
# Default value: 1
SpaceOutside = 1
```

**Comment by sfDesat** [comment]
This is fixed in 1.4.3. And the config file from SpaceWindows shouldn't matter, because it detects Celestial Tint and adjusts some settings relating to the sky accordingly.
